sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/FileServiceSasSignatureValues.java

```diff
@@ -61,13 +66,13 @@
     /**
      * Creates an object with empty values for all fields.
      */
-    FileServiceSasSignatureValues() {
+    public FileServiceSasSignatureValues() {
     }
 
     /**
      * Creates an object with the specified expiry time and permissions
      *
-     * @param expiryTime Time the SAS becomes valid
+     * @param expiryTime Time the SAS becomes invalid
      * @param permissions Permissions granted by the SAS
      */
     FileServiceSasSignatureValues(OffsetDateTime expiryTime, String permissions) {
```

Comment:
Does anything use this constructor?

Reply:
Not a soul.. and before this PR, no external user could create this class.